src/YesSql.Core/Session.cs

```diff
@@ -630,7 +630,7 @@ public void Dispose(bool disposing)
             {
                 try
                 {
-                    CommitOrRollbackTransaction();
+                    CommitOrRollbackTransactionAsync().ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
@sebastienros

Don't remember the details around this but are you sure about using `ConfigureAwait(false)` before `GetAwaiter()`?

Reply:
I think it just handles the error differently.

Do you see an issue?

Reply:
Would need more time to remember the details so not sure but reading again https://blog.stephencleary.com/2012/07/dont-block-on-async-code.html

For example this and that

> Using ConfigureAwait(false) to avoid deadlocks is a dangerous practice.

> The top-level methods, meanwhile, do require the context, so they cannot use ConfigureAwait(false)

I would not have used `.ConfigureAwait(false)` here but inside `CommitOrRollbackTransactionAsync()`, applied on the async call that it does.

Which may be useless if it is called asynchronously but maybe better for this synchronous usage.

But not sure ;)

Reply:
In the doubt I would not have used it at all.

Reply:
Okay. I'll remove it tomorrow before releasing the new version

Reply:
@jtkech what about in the other places like Session.Save and the SchemeBuilder?

Reply:
Idem for me

Reply:
Just to confirm, you suggest removing .ConfigureAwait(false) everywhere it's used? So .GetAwaiter().GetResult() only?

Reply:
Yes for now

Reply:
@jtkech done via https://github.com/sebastienros/yessql/pull/511 

Let me know if you need anything else modified